warehouse/query-core/src/test/java/datawave/query/iterator/logic/DColumnSummaryIteratorTest.java

```diff
@@ -0,0 +1,255 @@
+package datawave.query.iterator.logic;
+
+import static datawave.query.iterator.logic.DColumnSummaryIterator.VIEW_NAMES;
+import static datawave.query.iterator.logic.DColumnSummaryIterator.ONLY_SPECIFIED;
+import static datawave.query.iterator.logic.DColumnSummaryIterator.SUMMARY_SIZE;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.AbstractMap;
+import java.util.ArrayList;
+import java.util.Base64;
+import java.util.Collections;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.zip.GZIPOutputStream;
+
+import org.apache.accumulo.core.data.Key;
+import org.apache.accumulo.core.data.PartialKey;
+import org.apache.accumulo.core.data.Range;
+import org.apache.accumulo.core.data.Value;
+import org.apache.accumulo.core.iterators.IteratorEnvironment;
+import org.apache.accumulo.core.security.ColumnVisibility;
+import org.apache.hadoop.io.Text;
+import org.easymock.EasyMockRunner;
+import org.easymock.EasyMockSupport;
+import org.easymock.Mock;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import datawave.ingest.mapreduce.handler.ExtendedDataTypeHandler;
+import datawave.query.Constants;
+import datawave.query.iterator.SortedListKeyValueIterator;
+
+@RunWith(EasyMockRunner.class)
+public class DColumnSummaryIteratorTest extends EasyMockSupport {
```

Comment:
As with the iterator, consider renaming this to `ContentSummaryIteratorTest`

Reply:
done